src/pages/cart/cart.css

```diff
@@ -1,81 +1,107 @@
 
-.main{
-    background-color: var(--off-white);
-    height: 100vh;
+.cart-main {
+    display: grid;
+    grid-template-areas: 2fr 1fr;
+    grid-auto-flow: column;
+    margin: 1rem;
+    gap: 1rem;
+    justify-content: space-evenly;
 }
 
-.cart-body {
+.cart-body{
+    height: max-content;
     display: flex;
-    gap: 5rem;
-    justify-content: space-around;
-}
-.cart-title {
-    text-align: center;
-    font-size: 1.5rem;
-    font-weight: 600;
-    padding: 1rem;
+    flex-wrap: wrap;
+    gap: 10px;
+    width: 50vw
 }
 
-.card {
-    width: 20rem;
-    border: 1px solid var(--clr-black);
+.quantity{
+    align-self: center;
+    padding: 0.7rem 0.55rem;
 }
 
-.card-title{
-    text-align: center;
-}
-.card-sub-title{
-    text-align: center;
-    
-}
-
-.card-quantity{
+.cart-price-title{
+    font-weight: 600;
     text-align: center;
-    font-size: 1.2rem;
-}
-.card-quantity i {
-    cursor: pointer;
 }
 
 .cart-price-content{
-    display: flex;
+    width: 18em;
```

Comment:
use rem instead of em

Reply:
Thanks for the suggestion. I used em for relative responsiveness.